account_invoice_fiscal_position_update/README.rst

```diff
@@ -38,10 +41,13 @@ Credits
 Contributors
 ------------
 
+* Roel Adriaans <roel@road-support.nl>
```

Comment:
Please put you in last position, as this history is supposed to be from beginning to recent contributions